src/current/_includes/releases/v23.2/v23.2.21.md

```diff
@@ -0,0 +1,22 @@
+## v23.2.21
+
+Release Date: March 6, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-21-sql-language-changes">SQL language changes</h3>
+
+- Since v23.2, table statistics histograms have been collected for non-indexed `JSON` columns. Histograms are no longer collected for these columns if the `sql.stats.non_indexed_json_histograms.enabled` cluster setting is set to `false`. This reduces memory usage during table statistics collection, for both automatic and manual collection via `ANALYZE` and `CREATE STATISTICS`. [#140144][#140144]
+
+<h3 id="v23-2-21-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where under rare circumstances draining a node could fail with `some sessions did not respond to cancellation within 1s`. [#139479][#139479]
+- Fixed a bug that prevented the `CREATE` statement for a routine from being shown in a statement bundle. This happened when the routine was created on a schema other than `public`. The bug has existed since v23.1. [#136131][#136131]
+- Fixed a bounded memory leak that could previously occur when evaluating some memory-intensive queries via the vectorized engine. The leak has been present since v20.2. [#139098][#139098]
+- Previously, in changefeeds using CDC queries and the Parquet format, the output would include duplicate columns when it contained a user-defined primary key. Now, the columns are de-duplicated columns in the output when writing to Parquet. [#140380][#140380]
```

Comment:
typo: "now the columns are deduplicated columns" 

suggest removing second "columns"

Reply:
Thanks for the catch!